### PR TITLE
Style public (logged out) library header (prelimary).

### PR DESCRIPTION
**Pull request by yipingliao**
DO NOT MERGE Awaiting consultation with Danny and new Kifi logo asset.

@andrewconner I've still got a couple of things to check with Danny and need the new new Kifi logos, but wanted to put this in as a pull right now.

### CODE REVIEW

File: kifi-backend/modules/shoebox/angular/src/layout/main/main.styl [hunk 1]
```diff
@@ -10,6 +10,9 @@
     //border-right: 1px solid #dbdee3
     box-sizing: border-box
 
+.kf-logged-out .kf-main
+  width: 100%
```
Comment (yipingliao) [inline kifi-backend/modules/shoebox/angular/src/layout/main/main.styl:14]: Adding this allows the rendering to get the wider width in the beginning. That was causing problems with displaying the followers' pictures (with an initial narrow width, no follower pics were ever being displayed).

File: kifi-backend/modules/shoebox/angular/src/libraries/libraryCard.js [hunk 2]
```diff
@@ -21,6 +21,7 @@ angular.module('kifi')
         //
         // Scope data.
         //
+        scope.isUserLoggedOut = $rootScope.userLoggedIn === false;
```
Comment (yipingliao) [inline kifi-backend/modules/shoebox/angular/src/libraries/libraryCard.js:24]: I copied this from `dev.html` - using triple equals to be consistent; not sure if truthy/falsy test is adequate.

File: kifi-backend/modules/shoebox/angular/src/layout/header/publicHeader.tpl.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    <div class="kf-header-left">
+      <a href="/">
+        <span class="reco-icon svg-reco-icon"></span>
+        <img class="kf-header-logo" src="/img/kifi-logo-color.png">
+        <span class="public-header-tagline">Connecting people with knowledge</span>
+      </a>
+    </div>
+
+    <div class="kf-header-right">
+      <a href="//www.kifi.com/signup" class="public-header-link">Join Kifi</a>
```
Comment (yipingliao) [inline kifi-backend/modules/shoebox/angular/src/layout/header/publicHeader.tpl.html]: Keep the `kifi.com` hostname, or use `/signup` only? I needed the hostname for dev, but am not sure if this good practice for prod.

**Comment by andrewconner** [comment]
Wahoo!

File: kifi-backend/modules/shoebox/angular/src/layout/header/publicHeader.tpl.html (hunk 3 above)
Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/src/layout/header/publicHeader.tpl.html]: Yeah, I've been torn. For signup/auth, we actually do need the kifi.com domain.

Comment (2is10) [inline kifi-backend/modules/shoebox/angular/src/layout/header/publicHeader.tpl.html]: Could we use `env.navBase` instead? It’s intended specifically for cases when we need to navigate to a Play server route (on either dev.ezkeep.com:9000 or else www.kifi.com).

Comment (yipingliao) [inline kifi-backend/modules/shoebox/angular/src/layout/header/publicHeader.tpl.html]: Yes, thank you. Will do.

Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/src/layout/header/publicHeader.tpl.html]: Thanks @2is10. Yeah that's best.

**Comment by yipingliao** [comment]
Closing to resolve merge conflicts.

**Pull request closed without merging**